Behavioral.Automation.Template/Behavioral.Automation.Template.Bindings/Behavioral.Automation.Template.Bindings.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Maybe use the latest? 6.0?

Reply:
I think it would be nice to check compatibility between all of framework dependencies with 6th version of dotnet first

Reply:
@vladimir-s-quantori  Don't framework already can work with 6.0? Anyways I'll check and update

Reply:
I meant .net 6 support by other packages we reference here :)

Reply:
Done, everything works fine with .NET 6